### PR TITLE
Commit to create a sandbox for test.

### PR DESCRIPTION
**Pull request by SebastienBotte**
What a wonderful pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,4 +11,7 @@ Features:
 - Interact with customers
 - You can have superpower that helps you install more appliances and quicked etc...
 
-Fanfaronade ++
\ No newline at end of file
+I add some text here.
```
Comment (neilgibbs) [inline README.md:14]: I can see that
Thanks for your participation to my beautiful project

**Review by neilgibbs — APPROVED** [review]
All good !

File: README.md [hunk 2]
```diff
@@ -11,4 +11,7 @@ Features:
 - Interact with customers
 - You can have superpower that helps you install more appliances and quicked etc...
 
-Fanfaronade ++
```
Comment (vlandema) [inline README.md:14]: Should have kept this line 🕵️

**Review by vlandema — CHANGES_REQUESTED** [review]
This is wrong

**Pull request merged**